doc/ref_model/chapters/chapter03.md

```diff
@@ -98,17 +98,25 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 
 <a name="3.2.3"></a>
 ### 3.2.3 Storage
-A block device of a certain size for persisting information which can be created and dynamically attached to/detached from a virtual compute. A storage device resides in a tenant context and exists independently from any compute host.
 
-_**Example**: an OpenStack cinder volume._
+A workload can request different types of storage based on data longevity: persistent or ephemeral storage.
+Persistent storage outlives the compute instance whereas ephemeral storage is linked to compute instance lifecycle.
 
-| Attribute      | Description                                                              |
-|----------------|--------------------------------------------------------------------------|
-| `name`         | name of storage resources                                                |
-| `size`         | size of disc in GB                                                       |
-| `attachments`  | list of compute hosts to which the device is currently attached          |
-| `acceleration` | key/value pairs for selection of the appropriate acceleration technology |
-| `metadata`     | key/value pairs for selection of the appropriate redundancy domain       |
+There are multiple storage performance requirements such as latency, IOPS and capacity. For example, a workload may require one of its storage device to provide low latency, high IOPS and very large/huge storage capacity (terabytes of data).
+Low Latency storage is for workloads which have strong constraints on the time to access the storage.
+High IOPS oriented storage is for workloads requiring lots of read/write actions.
+Capacity oriented storage is for workloads that need lots of volumetry without strong perfomance constraints.
+
+Storage resources have the following attributes:
```

Comment:
do we also need a device type (HDD, SSD, ...)?
do we need to include SDS (e.g., ceph)?

Reply:
@pgoyal01 I think the chapter 3 must keep only the modelling aspect without going into technologies. For me, it is not the good place to describe the device types and SDS. 
But you are right these topics must be addressed.  Can it be integrated into chapter 4, adding a section under 4.1.4 internal Infra capabilities?

Reply:
I agree with @karinesevilla that such details would be better to have in Chapter 4.  @pgoyal01, if you agree, I would ask @karinesevilla to raise an issue to create a subsection 0f 4.1.4 to deal with the technology aspects of storage

Reply:
@walterkozlowski @karinesevilla I agree with the approach to raise an issue and insert this in Ch4. Thanks